README.md

```diff
@@ -355,6 +355,19 @@ Mode for config_file.
 
 - *Default*: '0640'
 
+config_file_contents
+--------------------
+Limits that should be placed into limits.conf
+
+- *Default*: undef
+
+config_file_source
```

Comment:
I don't think we need to support source, since we support contents, unless you really want this. Removing will make testing easier.